src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todos: Todo[];
+  setSelectedTodo: (todo: null) => void;
+  selectedTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todos,
+  setSelectedTodo,
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
Why default value is `null` instead of `{}`? If there were an object, you would always have one data type